core/selection-controller.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @module core/selection-controller
+ */
+var Montage = require("montage").Montage;
+
+/**
+ * @class SelectionController
+ * @extends external:Montage
+ */
+exports.SelectionController = Montage.specialize(/* @lends SelectionController.prototype */ {
+    constructor: {
+        value: function SelectionController() {
+            this.super();
+        }
+    },
+
+    /**
+     * @type {boolean}
+     */
+    isMultipleSelectionAllowed: {
+        value: false
+    },
+
+    /**
+     * @type {[object]}
+     */
+    selection: {
+        value: []
+    }
+
+});
```

Comment:
How about adding a comment in the Class JSDoc saying this is just a placeholder with the expected API, not anyting functional.